api/lib/application/sessions/finalized-session-controller.js

```diff
@@ -1,9 +1,15 @@
 const usecases = require('../../domain/usecases');
-const finalizedSessionSerializer = require('../../infrastructure/serializers/jsonapi/finalized-session-serializer');
+const publishableSessionSerializer = require('../../infrastructure/serializers/jsonapi/publishable-session-serializer');
+const sessionWithRequiredActionSerializer = require('../../infrastructure/serializers/jsonapi/session-with-required-action-serializer');
 
 module.exports = {
   async findFinalizedSessionsToPublish() {
```

Comment:
Renommer aussi le controller et le use case "SessionsToPublish" en "PublishableSessions" pour être cohérent?

Reply:
A choisit je prefère toPublish (à publier) plutôt que publishable (publiable) qui revèle davantage l'intention (i.e. celle de publier) du coup le publishable est juste là comme adaptation pour le front, où il est plus facile d'avoir le nom en fin publishable-session qu'au début session-to-be-published pour des histoire de "s" en fin 🙄 du coup je serai d'avis de laisser comme ça.